src/org/purefn/river/serdes/nippy.clj

```diff
@@ -0,0 +1,74 @@
+(defmacro ^:private compile-if
+  "Evaluate `exp` and if it returns logical true and doesn't error, expand to
+  `then`.  Else expand to `else`.
+  (compile-if (Class/forName \"java.util.concurrent.ForkJoinTask\")
+    (do-cool-stuff-with-fork-join)
+    (fall-back-to-executor-services))"
+  [exp then else]
+  (if (try (eval exp)
+           (catch Throwable _ false))
+    `(do ~then)
+    `(do ~else)))
+
+(compile-if
+ (do (require 'taoensso.nippy)
+     true)
+
+ (do
```

Comment:
White space is this `cool-stuff` makes it harder to distinguish from the `fall-back`.